### PR TITLE
Added azure/metric-alerts.json

### PR DESCRIPTION
**Pull request by jack-cloud-platform**

### CODE REVIEW

File: azure/metric-alerts.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  "parameters": {
+    "metricAlertsArray": {
+      "type": "array"
+    },
+    "actionGroupResourceId": {
+      "type": "string"
+    }
+  },
+  "variables": {
+    "deploymentUrlBase": "https://raw.githubusercontent.com/SkillsFundingAgency/das-platform-building-blocks/DASD-8884-add-alert-rule-building-block/templates/"
```
Comment (NickGraham101) [inline azure/metric-alerts.json:13]: Using branch URL

File: azure/metric-alerts.json [hunk 2]
```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2018-05-01/subscriptionDeploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "metricAlertsArray": {
+      "type": "array"
```
Comment (NickGraham101) [inline azure/metric-alerts.json:6]: Can we add a description with an example of the object that goes into the array?

File: azure/metric-alerts.json [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+            "value": "[replace(concat(parameters('metricAlertsArray')[copyIndex()].appInsightsName, ' - ', parameters('metricAlertsArray')[copyIndex()].metricName), '/', '-')]"
+          },
+          "alertDescription": {
+            "value": "[concat(parameters('metricAlertsArray')[copyIndex()].metricName, ' ', parameters('metricAlertsArray')[copyIndex()].metricOperator, ' ', parameters('metricAlertsArray')[copyIndex()].metricThreshold)]"
+          },
+          "alertSeverity": {
+            "value": 0
+          },
+          "isEnabled": {
+            "value": true
```
Comment (NickGraham101) [inline azure/metric-alerts.json:38]: Do we want it enabled in every environment?

Comment (jack-cloud-platform) [reply azure/metric-alerts.json]: Added areMetricAlertsEnabled parameter, a deployment to PP with disabled alerts before a deployment to PRD with enabled alerts sounds good.

**Review by NickGraham101 — APPROVED** [review]

**Pull request merged**